publish-mqtt/src/main.rs

```diff
@@ -238,16 +238,18 @@ async fn requests(requests_tx: Sender<Request>, device_base: &str) -> Result<(),
             if let Some((temperature, humidity, battery_voltage, battery_percent)) =
                 decode_value(&value)
             {
+                let mac_address = device.get_address()?;
+                let name = sensor_names.get(&mac_address).unwrap_or(&mac_address);
                 println!(
-                    "{} Temperature: {:.2}ºC Humidity: {:?}% Battery {} mV ({} %)",
+                    "{} ({}) Temperature: {:.2}ºC Humidity: {:?}% Battery {} mV ({} %)",
```

Comment:
might be better to put the name last, so that the readings show up in columns in the output

Reply:
Done.